src/dotnet/APIView/APIViewWeb/Controllers/PullRequestController.cs

```diff
@@ -25,38 +25,21 @@ public PullRequestController(PullRequestManager pullRequestManager, ILogger<Auto
                 
         [HttpGet]
         public async Task<ActionResult> DetectApiChanges(

```

Comment:
Is removing this a breaking change for any existing users? Or will the default parameters in the overload below work for the rest endpoint?

Reply:
Nope. It's covered by overloaded method. I have verified for existing PR.